com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -274,6 +282,29 @@ public float GetCurrentScale()
             return (m_Enabled && m_CurrentCameraRequest) ? m_CurrentFraction : 1.0f;
         }
 
+        /// <summary>
+        /// Gets the resolved scale percentages.
+        /// Note: this function is pure, and has no side effects.
+        /// </summary>
+        /// <returns>The resolved drs scales (from 0 to 1, for x and y). These might change depending on what the hardware supports.</returns>
+        public Vector2 GetResolvedScale()
```

Comment:
Do we need this API as public? The less we have public the better is for us as it will allow us to easily modify with time. I think with the no-side effect scale thingie we don't need this as public

Reply:
private now, done!